scipy/constants/_codata.py

```diff
@@ -1495,6 +1532,95 @@
 Wien wavelength displacement law constant                   2.897 771 955... e-3     (exact)                  m K
 W to Z mass ratio                                           0.881 53                 0.000 17                   """
 
+
+def exact2018(exact):
+    # SI base constants
+    c = exact['speed of light in vacuum']
+    h = exact['Planck constant']
+    e = exact['elementary charge']
+    k = exact['Boltzmann constant']
+    N_A = exact['Avogadro constant']
+
+    # Conventional electrical unit
+    K_J90 = exact['conventional value of Josephson constant']
+    K_J = 2 * e / h
+    R_K90 = exact['conventional value of von Klitzing constant']
+    R_K = h / e**2
+    V_90 = K_J90 / K_J
+    ohm_90 = R_K / R_K90
+    A_90 = V_90 / ohm_90
+
+    # Wien law numerical constants
+    alpha_W = (3 + lambertw(-3 * math.exp(-3))).real
+    x_W = (5 + lambertw(-5 * math.exp(-5))).real
```

Comment:
The original PR defined numerical values. Fortunately, `lambertw` gives exactly the same result as `mpmath`, but it would probably be safer to pre-calculate these.